public/app/stores/gateway/gateway-store.js

```diff
@@ -9,24 +11,45 @@ module.exports = Reflux.createStore({
     this.onFetchGateway();
   },
 
+  getSetupAppUrl: function getSetupAppUrl () {
+
+    function isOSX () {
+      var userAgentString = navigator.userAgent;
+      var environment = platform.parse(userAgentString);
+      return /OS X/.test(environment.os.family);
+    }
+
+    if (isOSX(platform)) {
+      return window.yetu.config.setupDownloadUrlMac;
+    } else {
```

Comment:
Reasoning here is that "all other platforms are Windows" because we don't have any fallback for the platforms that are not OS X or Windows. So perhaps a link to the Windows app is better than nothing :smile_cat: